.github/workflows/main.yml

```diff
@@ -21,7 +21,7 @@ env:
   # If you change this value, please change it in the following files as well:
   # /.travis.yml
   # /Dockerfile
-  GO_VERSION: 1.17.2
+  GO_VERSION: 1.17.12
```

Comment:
not: why not 1.18?

Reply:
There are some unresolved problems with the linter and 1.18. We need to solve them in lnd first and then do the same in all other projects.